RELEASE.md

```diff
@@ -0,0 +1,39 @@
+Release Steps
+=============
+
+1. Ensure tests are passing on [Circle CI](https://app.circleci.com/pipelines/github/instana/go-sensor)
+2. Make sure your local `master` branch is up-to-date:
+
+   ```bash
+   git checkout master && git pull --rebase
+   ```
+3. If you're about to release a new version of the main module, bump the package version in [`version.go`](./version.go), commit & push the
```

Comment:
@andrewslotin, what do you think if we will define a template for this commit message? It will allow to set up some githooks to prevent mistakes with tagging.

Reply:
Erm, I'm not quite sure how a template would prevent tagging issues. Could you elaborate?